index.css

```diff
@@ -25,7 +25,7 @@ section:target {
 }
 
```

Comment:
- [ ] Kindly make your pull request title more descriptive such as **Awesome books: refactor to use JavaScript classes** instead of `Update index.css` to make your pull request professional 😊
- [ ] Kindly make your readme more descriptive by adding the correct data such as title, summary, Getting Started, Future Features, Key Features, Tech Stack 👍
- [ ] Kindly use the correct linter setup for your application because Lighthouse and Webhint are missing from your linter setup 🤔